source/net/ServerSocket.ooc

```diff
@@ -127,7 +130,7 @@ ServerSocket: class extends Socket {
 				shutdown(conn sock descriptor, SHUT_RDWR)
 			}
 			conn close()
-			if ((conn && ret) as Bool)
+			if (ret as Bool)
```

Comment:
Right, after `conn close()`, this memory is forfeited?
You don't need the cast to `Bool` anymore since `f` returns `Bool`.